### ISSUE TITLE
Ability to Edit Title, Caption, Description, and Alt Text of Images in WordPress App Media Library Without Jetpack

### ISSUE THREAD

**Issue by mdrockwell** [wordpress-mobile/WordPress-iOS#20726]
### Expected behavior
If you're connecting to a site without the Jetpack plugin, you should be able to update the Title, Caption, Description, and Alt Text for media within the media library.

### Actual behavior
If you're connected to a site without the Jetpack plugin, you can view the Title, Caption, Description, and Alt Text, but there is no way to update it.

### Steps to reproduce the behavior
1. Connect to a site in the WordPress app using your WP Admin credentials (without Jetpack).
2. Navigate to the media library and tap on an image.
3. Note that the Title, Caption, Description, and Alt Text cannot be changed.

##### Tested on iPhone 13 Pro, iOS 16.4.1, WordPress iOS 22.4

**Comment by mdrockwell** [comment]
Another request regarding this from the forums:

https://wordpress.org/support/topic/images-cannot-be-edited-without-jetpack-plugin/

**Comment by tiagomar** [comment]
Similar issue on Android: https://github.com/wordpress-mobile/WordPress-Android/issues/18514

**Comment by marijn390** [comment]
Can someone indicate when this will be resolved? This bug is very annoying.
